microsoft/transfer.py

```diff
@@ -0,0 +1,486 @@
+        "File glob for the training dataset. If the files refer to Frame Level "
+        "features (i.e. tensorflow.SequenceExample), then set --reader_type "
+        "format. The (Sequence)Examples are expected to have 'rgb' byte array "
+        "sequence feature as well as a 'labels' int64 context feature.")
+    flags.DEFINE_string("feature_names", "mean_rgb", "Name of the feature "
+                        "to use for training.")
+    flags.DEFINE_string("feature_sizes", "1024",
+                        "Length of the feature vectors.")
+
+    # Model flags.
+    flags.DEFINE_bool(
+        "frame_features", False,
+        "If set, then --train_data_pattern must be frame-level features. "
+        "Otherwise, --train_data_pattern must be aggregated video-level "
+        "features. The model must also be set appropriately (i.e. to read 3D "
+        "batches VS 4D batches.")
+    flags.DEFINE_string(
+        "model", "LogisticModel",
+        "Which architecture to use for the model. Models are defined "
+        "in models.py.")
+    flags.DEFINE_bool(
+        "start_new_model", False,
+        "If set, this will not resume from a checkpoint and will instead create a"
+        " new model instance.")
+
+    # Training flags.
+    flags.DEFINE_integer("num_gpu", 1,
+                         "The maximum number of GPU devices to use for training. "
+                         "Flag only applies if GPUs are installed")
+    flags.DEFINE_integer("batch_size", 1024,
+                         "How many examples to process per batch for training.")
+    flags.DEFINE_string("label_loss", "CrossEntropyLoss",
+                        "Which loss function to use for training the model.")
+    flags.DEFINE_float(
+        "regularization_penalty", 1.0,
+        "How much weight to give to the regularization loss (the label loss has "
+        "a weight of 1).")
+    flags.DEFINE_float("base_learning_rate", 0.01,
+                       "Which learning rate to start with.")
+    flags.DEFINE_float("learning_rate_decay", 0.95,
+                       "Learning rate decay factor to be applied every "
+                       "learning_rate_decay_examples.")
+    flags.DEFINE_float("learning_rate_decay_examples", 4000000,
+                       "Multiply current learning rate by learning_rate_decay "
+                       "every learning_rate_decay_examples.")
+    flags.DEFINE_integer("num_epochs", None,
+                         "How many passes to make over the dataset before "
+                         "halting training.")
+    flags.DEFINE_integer("max_steps", None,
+                         "The maximum number of iterations of the training loop.")
+    flags.DEFINE_integer("export_model_steps", 1000,
+                         "The period, in number of steps, with which the model "
+                         "is exported for batch prediction.")
+
+    # Other flags.
+    flags.DEFINE_string("base_checkpoint", "model.ckpt-3425",
```

Comment:
Any particular reason it defaults to `model.ckpt-3425`?

Reply:
No reason, this was left as default from Mihaela. Should I change it to model.ckpt-0?

Reply:
yea, ckpt-0 makes sense